Library/Formula/ttf2pt1.rb

```diff
@@ -4,9 +4,43 @@ class Ttf2pt1 < Formula
   url "https://downloads.sourceforge.net/ttf2pt1/ttf2pt1-3.4.4.tgz"
   sha256 "ae926288be910073883b5c8a3b8fc168fde52b91199fdf13e92d72328945e1d0"
 
+  patch :DATA
```

Comment:
Unfortunately this is hopeless. The last release of the tool was in 2003 and unmaintained now:
<http://ttf2pt1.sourceforge.net/download.html>

Without the patch I get the error `/usr/local/include/freetype/freetype.h:24:2: error: "'ft2build.h' hasn't been included yet!"`. The patch is needed to fix the freetype headers for its changed API and v2.